commands/volumes/volume-option.go

```diff
@@ -58,6 +58,10 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		restutils.SendHTTPError(w, http.StatusBadRequest, fmt.Sprintf("invalid option specified: %s", err.Error()))
 		return
 	}
+	// Update the options into volinfo
```

Comment:
Already this code is available in line 106. Looks like some other issue while storing in volinfo

Reply:
Correct, I overlooked this code. Let me figure it out where the problem is.